### PR TITLE
Add Quantity & VatRate to feedspec

### PR DESCRIPTION
**Pull request by MrGoumX**

### CODE REVIEW

File: source/localizable/feedspec/index.el.html.md.erb [hunk 1]
```diff
@@ -171,6 +171,7 @@ ImageBot v1".
 [Additional Image Link](#additional-image-link)        | String  | <span class="mono">400</span>  | Όχι<b><sup>1</sup></b>| `https://www.someurl.com/images/322/front_view.jpg`
 [Category Name](#category-name)                        | String  | <span class="mono">250</span>  | Ναι         | `Κινητά τηλέφωνα`
 [Price](#price)                                        | Decimal | <span class="mono">- (έως 2 δεκαδικά ψηφία)</span>    | Ναι         | `23.22`
+[Vat Rate](#vat-rate)                                  | Decimal | <span class="mono">-</span>    | Όχι         | `24.00`
```
Comment (mrwhizzy) [inline source/localizable/feedspec/index.el.html.md.erb:174]: we can also point out that this field accepts precision 2 (same as `price` above)


```suggestion
[Vat Rate](#vat-rate)                                  | Decimal | <span class="mono">- (έως 2 δεκαδικά ψηφία)</span> | Όχι         | `24.00`
```

File: source/localizable/feedspec/index.el.html.md.erb [hunk 2]
```diff
@@ -381,6 +383,24 @@ thumbnail.
 
 > Το πεδίο τιμής μπορεί να δεχτεί μέχρι 2 δεκαδικά ψηφία.
 
+#### Vat Rate
```
Comment (mrwhizzy) [inline source/localizable/feedspec/index.el.html.md.erb:386]:
```suggestion
#### VAT Rate
```

File: source/localizable/feedspec/index.en.html.md.erb [hunk 3]
```diff
@@ -166,6 +166,7 @@ Name                                                   | Type    | Length
 [Additional Image Link](#additional-image-link)        | String  | <span class="mono">400</span>  | No<b><sup>1</sup></b>| `https://www.someurl.co.uk/images/322/front_view.jpg`
 [Category Name](#category-name)                        | String  | <span class="mono">250</span>  | Yes      | `Mobile phones`
 [Price](#price)                                        | Decimal | <span class="mono">- (up to 2 decimal places)</span>    | Yes      | `23.22`
+[Vat Rate](#vat-rate)                                  | Decimal | <span class="mono">-</span>    | No       | `24.00`
```
Comment (mrwhizzy) [inline source/localizable/feedspec/index.en.html.md.erb:169]:
```suggestion
[VAT Rate](#vat-rate)                                  | Decimal | <span class="mono">- (up to 2 decimal places)</span>    | No       | `24.00`
```

File: source/localizable/feedspec/index.en.html.md.erb [hunk 4]
```diff
@@ -345,6 +347,22 @@ It's the products retail price **including V.A.T**.
 
 > The price field accepts up to 2 decimal places.
 
+#### Vat Rate
```
Comment (mrwhizzy) [inline source/localizable/feedspec/index.en.html.md.erb:350]:
```suggestion
#### VAT Rate
```

File: source/localizable/feedspec/index.en.html.md.erb [hunk 5]
```diff
@@ -345,6 +347,22 @@ It's the products retail price **including V.A.T**.
...
+
+It's the product's VAT rate.
```
Comment (mrwhizzy) [inline source/localizable/feedspec/index.en.html.md.erb:352]:
```suggestion
It's the product's V.A.T. rate.
```

File: source/localizable/feedspec/index.el.html.md.erb [hunk 6]
```diff
@@ -381,6 +383,24 @@ thumbnail.
...
+
+Στο πεδίο αυτό μπορείτε να συμπληρώσετε τον Φ.Π.Α του προϊόντος ως ποσοστό.
```
Comment (mrwhizzy) [inline source/localizable/feedspec/index.el.html.md.erb:388]:
```suggestion
Στο πεδίο αυτό μπορείτε να συμπληρώσετε τον συντελεστή Φ.Π.Α του προϊόντος ως ποσοστό.
```

File: source/localizable/feedspec/index.el.html.md.erb [hunk 7]
```diff
@@ -381,6 +383,24 @@ thumbnail.
...
+
+Το πεδίο είναι προαιρετικό αλλά συνιστάται η χρήση του.
+
+> Το πεδίο τιμής μπορεί να δεχτεί μέχρι 2 δεκαδικά ψηφία.
```
Comment (mrwhizzy) [inline source/localizable/feedspec/index.el.html.md.erb:392]:
```suggestion
> Το πεδίο συντελεστή Φ.Π.Α. μπορεί να δεχτεί μέχρι 2 δεκαδικά ψηφία.
```

File: source/localizable/feedspec/index.el.html.md.erb [hunk 8]
```diff
@@ -381,6 +383,24 @@ thumbnail.
...
+
+> Το πεδίο δέχεται ακέραιες τιμές μεγαλύτερες ή ίσες του `0`.
```
Comment (mrwhizzy) [inline source/localizable/feedspec/index.el.html.md.erb:394]:
```suggestion
> Το πεδίο δέχεται τιμές στο διάστημα μεταξύ του `1` και του `100` ή ίσες του `0`.
```

File: source/localizable/feedspec/index.en.html.md.erb [hunk 9]
```diff
@@ -345,6 +347,22 @@ It's the products retail price **including V.A.T**.
...
+
+> The price field accepts up to 2 decimal places.
```
Comment (mrwhizzy) [inline source/localizable/feedspec/index.en.html.md.erb:354]:
```suggestion
> The V.A.T. rate field accepts up to 2 decimal places.
```

File: source/localizable/feedspec/index.en.html.md.erb [hunk 10]
```diff
@@ -345,6 +347,22 @@ It's the products retail price **including V.A.T**.
...
+
+> The quantity field only accepts integer values greater or equal to `0`.
```
Comment (mrwhizzy) [inline source/localizable/feedspec/index.en.html.md.erb:356]:
```suggestion
> This field accepts values within the range between `1` and` 100`, or equal to `0`.
```

**Review by mrwhizzy — APPROVED** [review]
other than my comments LGTM!

File: source/localizable/feedspec/index.el.html.md.erb (hunk 1 above)
Comment (MrGoumX) [reply source/localizable/feedspec/index.el.html.md.erb]: Done

File: source/localizable/feedspec/index.el.html.md.erb (hunk 2 above)
Comment (MrGoumX) [reply source/localizable/feedspec/index.el.html.md.erb]: Done

File: source/localizable/feedspec/index.en.html.md.erb (hunk 3 above)
Comment (MrGoumX) [reply source/localizable/feedspec/index.en.html.md.erb]: Done

File: source/localizable/feedspec/index.en.html.md.erb (hunk 4 above)
Comment (MrGoumX) [reply source/localizable/feedspec/index.en.html.md.erb]: Done

File: source/localizable/feedspec/index.en.html.md.erb (hunk 5 above)
Comment (MrGoumX) [reply source/localizable/feedspec/index.en.html.md.erb]: Done

File: source/localizable/feedspec/index.el.html.md.erb (hunk 6 above)
Comment (MrGoumX) [reply source/localizable/feedspec/index.el.html.md.erb]: Done

File: source/localizable/feedspec/index.el.html.md.erb (hunk 7 above)
Comment (MrGoumX) [reply source/localizable/feedspec/index.el.html.md.erb]: Done

File: source/localizable/feedspec/index.el.html.md.erb (hunk 8 above)
Comment (MrGoumX) [reply source/localizable/feedspec/index.el.html.md.erb]: Done

File: source/localizable/feedspec/index.en.html.md.erb (hunk 9 above)
Comment (MrGoumX) [reply source/localizable/feedspec/index.en.html.md.erb]: Done

File: source/localizable/feedspec/index.en.html.md.erb (hunk 10 above)
Comment (MrGoumX) [reply source/localizable/feedspec/index.en.html.md.erb]: Done

**Review by harrisred — APPROVED** [review]
LGTM 👍

**Pull request merged**